README.md

```diff
@@ -54,6 +54,109 @@ $ go get -u github.com/skroutz/mistry/cmd/mistry
 
 
 
+Examples
+--------------------------------------------------
+Here is a little preview of how building looks with mistry.
+
+### Bundler
```

Comment:
This could be a README under each project's directory.

Also, I believe we should rename it to `bundler-cache` so that it's more apparent what this is about.

Reply:
sounds good, updated